index.html

```diff
@@ -20,15 +20,19 @@
                 <a href="#"><p>Worash Aboche</p></a>  
             </div>
             <nav>
-               <ul class="nav-link">
-                  <li><a href="#portfolio" role="link"  aria-label="portfolio link">Portfolio</a></li>
-                  <li><a href="#about"  role="link" aria-label="about link">About </a></li>
-                  <li><a href="#contact"  role="link"  aria-label="contact link" >Contact</a></li>
-                  <li><a href="#"  role="link"  aria-label="message icon"><i class="fas fa-envelope"></i></a></li>
+               <ul class="nav-link hide">
+                  <li><a href="#portfolio"  id ="portfolio-link" role="link"  aria-label="portfolio link">Portfolio</a></li>
+                  <li><a href="#about-container"  id="about-link" role="link" aria-label="about link">About </a></li>
+                  <li><a href="#contact-conatiner" id="contact-link" role="link"  aria-label="contact link" >Contact</a></li>
+                  <li><a href="#"  role="link" id="message-envelope"  aria-label="message icon"><i class="fas fa-envelope "></i></a></li>
                </ul>
            </nav>
-            <div class="menu">
-               <img id ="hamberger" src ="./images/Icon-Menu.svg" alt="menu" > 
+            <div id ="hamburger" class="menu" >
```

Comment:
- [ ] Nice implementing the functionality, however, please try to stick to the Figma design by exporting gray the close button from Figma and the dividers between links  should also be gray as in the Figma design please try to fix that 

- [ ] Figma Design 
![image](https://user-images.githubusercontent.com/61790587/167691411-53e6b3df-ec16-4ec4-a11f-0183282ef6b8.png)

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/167691444-7f5fe8f5-77a7-4e1c-9509-103eb47a3b34.png)